index.js

```diff
@@ -1,14 +1,16 @@
 
```

Comment:
Why do we have an empty line here?

Reply:
sorry about the empty line... i don't know if it is a good practice. noticed it in many videos i watched.